src/main/java/org/digitalmediaserver/nsis/MakeMojo.java

```diff
@@ -293,13 +293,26 @@ private void validate() throws MojoExecutionException {
 			Path path = Paths.get(makensisExecutable);
 			makensisExecutable = path.toRealPath().toString();
 			if (autoNsisDir && isBlank(nsisDir)) {
+				if (Files.isSymbolicLink(path)) {
+					path = path.toRealPath();
+				}
 				path = path.getParent();
 				if (path != null) {
 					path = path.toRealPath();
 					Path fileName = path.getFileName();
 					if (fileName != null && "bin".equals(fileName.toString().toLowerCase(Locale.ROOT))) {
 						path = path.getParent();
 					}
+
+					// macOS Homebrew!
+					if (OSTYPE == OSType.MACOS) {
+						if (path.toString().contains("/Cellar/")) {
```

Comment:
I'm not very familiar with homebrew or macOS, but doesn't this mean that makensis isn't "properly installed" (that it's a "keg-only"?). Would running "brew link" fix this and make this special adaptation superfluous?

Reply:
See https://stackoverflow.com/a/17016758

Reply:
Forget the above, I understand now I think. Brew doesn't put `makensis` in the `Bin` folder when building.

The question is it we could make this more generic, my guess is that other builds could have this folder structure as well.

Reply:
The question is how to "recognize" the "NSIS folder". I guess probing for the `Include` and/or `Plugins` folder could be a reasonable approach. If these aren't found after backing out of the `bin` folder, maybe `share/nsis` could be checked?

Reply:
So using `brew link` will add an `/usr/local/share/nsis` folder, so we still need something special for homebrew even if someone runs `brew link`. I will make some updates.